install_bun.sh

```
@@ -42,15 +42,15 @@ error() {
 }
 
 info() {
-  printf '%b\n' "${Dim}$*${Color_Off}"
+    echo -e "${Dim}$@ ${Color_Off}"
 }
 
 info_bold() {
-  printf '%b\n' "${Bold_White}$*${Color_Off}"
+    echo -e "${Bold_White}$@ ${Color_Off}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)